python/pdo/test/contract.py

```diff
@@ -149,14 +149,18 @@ def CreateAndRegisterEnclave(config) :
 
     global enclave
     global txn_dependencies
+    ledger_config = config.get('Sawtooth')
 
     # if we are using the eservice then there is nothing to register since
     # the eservice has already registered the enclave
     if use_eservice :
         enclaveclients = []
         try :
             for url in config['Service']['EnclaveServiceURLs'] :
-                enclaveclients.append(db.get_client_by_url(url))
+                # need to add this to the database so the replication
+                # module can find it later
+                einfo = db.add_by_url(ledger_config, url)
+                enclaveclients.append(einfo.client)
```

Comment:
Same comment as in test-request. Please see below

Reply:
same as above... those were your comments.
will remove.

Reply:
done